src/main/webapp/app/exam/participate/exam-participation.component.html

```diff
@@ -28,7 +28,7 @@
             @for (exercise of studentExam.exercises; track exercise; let i = $index) {
                 @if (exercise && exercise.studentParticipations && exercise.studentParticipations[0]) {
                     @if (pageComponentVisited[i]) {
-                        <div [hidden]="i !== activePageIndex" [id]="'exercise-' + exercise.id">
+                        <div style="margin-bottom: 30px" [hidden]="i !== activePageIndex" [id]="'exercise-' + exercise.id">
```

Comment:
We prefer classes over inline-css for these purposes (not sure if 2 is the correct value to choose from, but is should be some `mb-x` class)
```suggestion
                        <div class="mb-2" [hidden]="i !== activePageIndex" [id]="'exercise-' + exercise.id">
```

Reply:
`mb-4` would be 1.5rem. This equals at the moment 24px.